notebooks/MAST/dwarf_galaxy_search/dwarf_galaxy_search.ipynb

```diff
@@ -0,0 +1,832 @@
+{
```

Comment:
<p>Please revise this cell that we could not find the other MAST data for the leo star. But to show how to retrieve another target's spectrum, create another cell to retrieve what you found using the larger radius though it's not a nearby object. This is the duplicate step above but with the larger radius before.</p><p> </p><p>So retrieve that object in the new cell. Show its header. Also print out the important header information such as the target name, RA/Dec, telescope, instrument, filter, exposure time, etc before specifying the filter. Then keep the below cells to show the spectrum.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>